src/FxCop/Desktop.Analyzers/Core/Security/DoNotUseInsecureCryptographicAlgorithmsAnalyzer.cs

```diff
@@ -130,30 +153,33 @@ public void AnalyzeNode(SyntaxNodeAnalysisContext context)
 
                 if (type.IsDerivedFrom(this._cryptTypes.DES, baseTypesOnly: true))
                 {
-                    rule = DoNotUseDESSpecificRule;
+                    rule = CreateCA5351DiagnosticDescriptor(owningParentName, _cryptTypes.DES.Name);
```

Comment:
But the description still have placeholders. That can only be determined at the point where an issue is found. The descriptor consists description of it, doesn't it?